src/test/ui/editions/edition-keywords-2015-2018-expansion.stderr

```diff
@@ -7,8 +7,8 @@ LL |     produces_async! {}
    = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
 help: you can escape reserved keywords to use them as identifiers
    |
-LL | () => (pub fn r#async () { })
-   |               ^^^^^^^
+LL | r#async
```

Comment:
Maybe the test should be changed to have `pub fn async` on one line, so the diagnostic output makes more sense? Or am I missing something?

Reply:
Formatting the aux testfile seems to have no effect.

Reply:
I don't understand, that shouldn't be possible.

Reply:
Just saw your changes, `pub fn async` was already on one line, I guess I wasn't clear enough (I should've just checked).